src/app/components/Footer/Footer.tsx

```diff
@@ -1 +1,71 @@
-export {}
\ No newline at end of file
+import './Footer.scss'
+import logo from '../../../assets/icons/pageLogo.svg'
+import fb from '../../../assets/icons/fb.png'
+import tw from '../../../assets/icons/tw.png'
+import IN from '../../../assets/icons/IN.png'
+import insta from '../../../assets/icons/insta.png'
+import { Link } from 'react-router-dom'
+export const Footer = () => {
+    return (
+        <>
+        <footer>
+            <div className="main-interno">
+                <div className="interno">
+                    <div className="sub-content">
+                        <div className="logo">
+                            <img src={logo} alt="logo" />
+                        </div>
+                        <p className="text">
+                            It is a long established fact that a reader will be distracted
+                            lookings.
+                        </p>
+                    </div>
+                    <div className="icons">
+                        <button>
+                            <img src={fb} alt="fb" />
+                        </button>
+                        <button>
+                            <img src={tw} alt="tw" />
+                        </button>
+                        <button>
+                            <img src={IN} alt="in" />
+                        </button>
+                        <button>
+                            <img src={insta} alt="insta" />
+                        </button>
```

Comment:
seems like icons for social networks
in this case, they are not buttons, but links